test/itkTBBImageToImageFilterTest.cxx

```diff
@@ -73,7 +73,7 @@ class TBBImageToImageFilterHelper : public TBBImageToImageFilter< TInputImage, T
 
 } // itk
 
-int main(int argc, const char** argv)
+int itkTBBImageToImageFilterTest(int argc, const char** argv)
```

Comment:
You should replace `(int argc, const char** argv)` by `(int , char* [])` and then remove the `(void)argc; (void)argv;`